lib/vulcan.js

```diff
@@ -177,9 +177,13 @@ Vulcan.prototype = {
   },
 
   moveToBodyMatcher: dom5.predicates.AND(
+    dom5.predicates.NOT(
+        dom5.predicates.parentMatches(
+            dom5.predicates.hasTagName('template'))),
     dom5.predicates.OR(
       dom5.predicates.hasTagName('script'),
-      dom5.predicates.hasTagName('link')
+      dom5.predicates.hasTagName('link'),
```

Comment:
the OR includes the various attribute states for the script and link types we don't want to move.  I just added style because we do want to move those too.